website_sale_qty/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    'name': 'Website Sale - Price Tiers',
+    'summary': 'Add price tier radio buttons to product pages in website shop',
```

Comment:
Thanks for your contribution.

not blocking for me but why technical name and friendly name are different ?
If technical name is not explicit why not rename it ?

Reply:
I'm ok with a rename

Reply:
Fair enough. The name could definitely be more descriptive, so I've changed it to ``webite_sale_price_tiers``.